tests/v2/validation/upgrade/kubernetes_test.go

```diff
@@ -18,9 +19,11 @@ import (
 )
 
 const (
-	rke1KubeVersionCheck = "rancher"
-	rke2KubeVersionCheck = "rke2"
-	k3sKubeVersionCheck  = "k3s"
+	rke1KubeVersionCheck          = "rancher"
+	rke2KubeVersionCheck          = "rke2"
+	k3sKubeVersionCheck           = "k3s"
+	allMachineReadyTimeout        = 10 * time.Minute
+	allManagementNodeReadyTimeout = 30 * time.Minute
```

Comment:
I think we can add these 10-minute and 30-minute options to the extensions/default package and use these values from that package, applicable to all the time variables the PR has in these different packages